src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+    private final int index = random.nextInt(Colors.values().length);
```

Comment:
The variable 'index' should not be a field of the class since it's used only in the scope of the 'getRandomColor' method. Consider making it a local variable inside the method.